plugins/org.preesm.codegen.xtend/src/org/preesm/codegen/xtend/spider/visitor/SpiderCodegen.java

```diff
@@ -305,6 +310,7 @@ public String generateHeaderCode(final PiGraph pg, final SpiderConfig spiderConf
 
     /* Declare Include Files */
     append("#include <spider.h>\n\n");
+    append("#include <apolloAPI.h>\n\n");
```

Comment:
Isn't this highly likely to generate compile errors if the project does not use appolo ?

Reply:
You are right. That was part of a first version and should have been removed. I just removed it.